provisioning_templates/PXEGrub2/pxegrub2_default_local_boot.erb

```diff
@@ -7,5 +7,7 @@ model: ProvisioningTemplate
 
 set default=<%= global_setting("default_pxe_item_local", "local") %>
 set timeout=20
+echo Default PXE local template entry is set to '<%= global_setting("default_pxe_item_local", "local") %>'
```

Comment:
And this is just a information to the user what default item was set in Administer - Settings. I find this useful, also present in the global template.